docs/sql-reference/sql-statements/data-manipulation/insert.md

```diff
@@ -66,9 +66,9 @@ INSERT INTO test (c1, c2) VALUES (1, DEFAULT);
 INSERT INTO test (c1) VALUES (1);
 ```
 
-When no target column is specified, the columns are loaded in sequential order into the target table by default. Therefore, in the above example, the outcomes of the first and second SQL statements are the same.
+When no destination column is specified, the columns are loaded in sequential order into the destination table by default. Therefore, in the above example, the outcomes of the first and second SQL statements are the same.
 
-If there is a target column with no data inserted or data inserted using DEFAULT as the value, the column will use the default value as the loaded data. Therefore, in the above example, the outcomes of the third and fourth statements are the same.
+If there is a destination column with no data inserted or data inserted using DEFAULT as the value, the column will use the default value as the loaded data. Therefore, in the above example, the outcomes of the third and fourth statements are the same.
```

Comment:
If a destination column (with or without data inserted) uses DEFAULT as the value